code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2666,96 +2666,979 @@ Follow by example and make good judgement based on length which list to include
+
+	tat_forrest_right
+		name = "Tattoo (Forrest, Right Eye)"
+		icon_state = "tat_forrest_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hive
+		name = "Tattoo (Hive, Back)"
+		icon_state = "tat_hive"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_heart
+		name = "Tattoo (Heart, Chest)"
+		icon_state = "tat_heart"
+		body_parts = list("chest")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hunter_left
+		name = "Tattoo (Hunter, Left Eye)"
+		icon_state = "tat_hunter_l"
+		body_parts = list("head")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_hunter_right
+		name = "Tattoo (Hunter, Right Eye)"
+		icon_state = "tat_hunter_r"
+		body_parts = list("head")
+		species_allowed = list("Human","Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_jaeger_left
+		name = "Tattoo (Jaeger, Left Eye)"
+		icon_state = "tat_jaeger_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_jaeger_right
+		name = "Tattoo (Jaeger, Right Eye)"
+		icon_state = "tat_jaeger_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_joosten_left
+		name = "Tattoo (Joosten, Left Eye)"
+		icon_state = "tat_joosten_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_joosten_right
+		name = "Tattoo (Joosten, Right Eye)"
+		icon_state = "tat_joosten_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_miller_left
+		name = "Tattoo (Miller, Left Eye)"
+		icon_state = "tat_miller_l"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_miller_right
+		name = "Tattoo (Miller, Right Eye)"
+		icon_state = "tat_miller_r"
+		body_parts = list("head")
+		species_allowed = list("Human", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_montana_left
+		name = "Tattoo (Montana, Left Face)"
+		icon_state = "tat_montana_l"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	tat_montana_right
+		name = "Tattoo (Montana, Right Face)"
+		icon_state = "tat_montana_r"
+		body_parts = list("head")
+		species_allowed = list("Human")
+
+	tat_nightling
+		name = "Tattoo (Nightling, Back)"
+		icon_state = "tat_nightling"
+		body_parts = list("chest")
+		species_allowed = list("Human", "Skrell", "Tajara", "Zhan-Khazan Tajara", "M'sai Tajara", "Unathi")
+
+	tat_pawsocks
```

Comment:
This should be restricted to tajara only, this is going to create alot of problems with catgirl/foxgirl characters

Reply:
If it was named something else, do you think that'd lead people into not doing that?

Reply:
I guess we can try it and if it becomes a issue remove it